x/crisis/params.go

```diff
@@ -0,0 +1,35 @@
+package crisis
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/params"
+)
+
+// Default parameter namespace
+const (
+	DefaultParamspace = ModuleName
+)
+
+var (
+	// key for constant fee parameter
+	ParamStoreKeyConstantFee = []byte("ConstantFee")
+)
+
+// type declaration for parameters
+func ParamKeyTable() params.KeyTable {
+	return params.NewKeyTable(
+		ParamStoreKeyConstantFee, sdk.Coin{},
+	)
+}
+
+// GetConstantFee get's the constant fee from the paramSpace
+func (k Keeper) GetConstantFee(ctx sdk.Context) sdk.Coin {
+	var constantFee sdk.Coin
+	k.paramSpace.Get(ctx, ParamStoreKeyConstantFee, &constantFee)
+	return constantFee
+}
+
+// GetConstantFee set's the constant fee in the paramSpace
+func (k Keeper) SetConstantFee(ctx sdk.Context, constantFee sdk.Coin) {
+	k.paramSpace.Set(ctx, ParamStoreKeyConstantFee, &constantFee)
```

Comment:
We probably should not pass a pointer here

Reply:
oh okay I was confused as to how the param store works... will remove the pointer